<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUser</class>
 <widget class="QDialog" name="AddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add User</string>
  </property>
  <widget class="QLabel" name="lblMessage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblUser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>User</string>
   </property>
   <property name="buddy">
    <cstring>leUsername</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblPassword">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="buddy">
    <cstring>lePassword</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblPassword_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
   <property name="buddy">
    <cstring>lePassword_2</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUsername">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leUsername</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>lePassword_2</tabstop>
  <tabstop>btnReturn</tabstop>
  <tabstop>btnAdd</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>AddUser</receiver>
   <slot>onAddUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leUsername</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddUser</receiver>
   <slot>onEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePassword</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddUser</receiver>
   <slot>onEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePassword_2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddUser</receiver>
   <slot>onEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReturn</sender>
   <signal>clicked()</signal>
   <receiver>AddUser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddUser()</slot>
  <slot>onEdit()</slot>
 </slots>
</ui>
